src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.test.tsx

```diff
@@ -14,30 +14,72 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { createContext } from 'react';
+import { Provider } from 'react-redux';
```

Comment:
So, to clarify, `BlastInputSequence` component no longer needs redux? Then all redux code should be removed from the test.

Reply:
Good spot, i forgot about this after i changed the place of the context. I dont need to make any changes to the test file then and i have removed all my changes.

Reply:
Done